src/selic-entries.spec.ts

```diff
@@ -0,0 +1,5 @@
+import { selicEntries } from "./selic-entries";
```

Comment:
Hey, use single quotes whenever possible, ok?

```suggestion
// my fancy changes
import { selicEntries } from './selic-entries';
```